recipes/cassiopee/build.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+export C_INCLUDE_PATH=${PREFIX}/include:$C_INCLUDE_PATH
+export CPLUS_INCLUDE_PATH=${PREFIX}/include:$CPLUS_INCLUDE_PATH
+export LIBRARY_PATH=${PREFIX}/lib:$LIBRARY_PATH
+export LD_LIBRARY_PATH="${PREFIX}/lib"
+
+export LDFLAGS="-L${PREFIX}/lib"
```

Comment:
prefer `export LDFLAGS="$LDFLAGS ..."` for *FLAGS variables to append rather than overwrite

Reply:
done